### PR TITLE
Website made changes to maize...

### PR DESCRIPTION
**Pull request by aakarshandhakal**

### CODE REVIEW

**Comment by lewiskan** [comment]
@aakarshandhakal Looks good to me, how was the process?

**Comment by aakarshandhakal** [comment]
Process was smooth, please merge :)

File: .gitignore [hunk 1]
```diff
@@ -1 +1,3 @@
 .DS_Store
+node_modules
+website/.DS_Store
```
Comment (lewiskan) [inline .gitignore:3]: @aakarshandhakal Did this website/ trick work?

**Comment by aakarshandhakal** [comment]
No but lets use it for now...

**Comment by lewiskan** [comment]
+1

**Pull request merged**